apps/molgenis-components/src/components/forms/InputBoolean.vue

```diff
@@ -23,10 +25,12 @@ import BaseInput from "./baseInputs/BaseInput.vue";
 import InputRadio from "./InputRadio.vue";
 
 export default {
+  name: "InputBoolean",
   components: { InputRadio },
   extends: BaseInput,
   props: {
     inplace: { type: Boolean, default: false },
+    isClearable: {type: Boolean, default: true}
```

Comment:
we actually in styleguide use the 'required' semantics for this. For all components then holds that 'clear' is only shown if not required. I think we should depart from that.

However: what does 'clear' mean with a boolean? Is that 'false' or 'null'. In some cases this difference really matters.

Reply:
I think you probably prefer (and I probably as well) to have less magic on the components. So I can imagine we want to split the 'isClearable' from the 'checkNotNull' option (which is now implicit in this 'required' statement which is maybe too much magic?)

Reply:
yes , so we keep the isClearable for now